src/components/common/nested-select.vue

```diff
@@ -155,4 +155,12 @@ export default class NestedSelect extends Vue {
 ::v-deep .v-list-item:has(.v-list-item__content.hide-me) {
   display: none;
 }
+//Removes empty spaces on nested select when the list is collapsed
+::v-deep .v-list-item {
+  min-height: auto;
+}
+//Hide and remove space for disabled list items and makes the nested select work for firefox
```

Comment:
You can use `/* ... */` or `//` in the CSS, but we do like to have a space after the slashes.